### PR TITLE
added pull_request_template.md

### PR DESCRIPTION
**Pull request by PeculiarGoat**
Turns out that you get PR templates by adding this file under .github!  See [here](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository).

I've just copied the Relish PR template for now and figured we can discuss what needs to be added/changed/removed on this PR.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+ADO card: ☑️ [AB#XXXX](https://dev.azure.com/S72/SHIFT72/_boards/board/t/Delivery%20team/Stories/?workitem=XXXX)
```
Comment (stajs) [inline .github/pull_request_template.md:1]: Nit: This opens the board and then the card, prefer a direct link to the card. Also, I think we can just `AB#XXXX` for it to autolink.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+Elab notes/AC: 📃 [Google Docs](https://docs.google.com)
```
Comment (stajs) [inline .github/pull_request_template.md:3]: Perhaps something to encourage cross-team collab? e.g. "Has this been discussed with delivery team?"

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+Designs: 
+- 📱 [Mobile](http://www.zeplin.io/????)
+- 💊 [Tablet](http://www.zeplin.io/????)
+- 🖥 [Desktop](http://www.zeplin.io/????)
+
+## Description of work
+This is a description of the feature or bug, including any detail not in the ADO card that is required to understand the work.
```
Comment (stajs) [inline .github/pull_request_template.md:11]: I'm going to suggest that ADO cards are usually a bit lacking, we should be making it easy for reviewers and require a decent description here.

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+### Any PRs dependent on this one
+
+### Affected Clients
+ - All clients who use Kibble
+ - Client 1
+ - Client 2
+
+## Tests
+📈 Coverage increased from XX% to YY%
```
Comment (stajs) [inline .github/pull_request_template.md:33]: 😬

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+## Checklist
```
Comment (stajs) [inline .github/pull_request_template.md:35]: Need thought in to our checklist.

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+- Edge case 1
+- Caveat 2
+- Known issue 3
+
+## Config settings/Toggles required for the feature to work
+- Config setting 1
+
+## Related PRs 
+
+### Any PRs which this PR depends on
```
Comment (stajs) [inline .github/pull_request_template.md:23]: On dependencies, need to highlight Relish.

**Review by stajs — CHANGES_REQUESTED** [review]
A good start, probably just talk it over after standup to iron out the wrinkles.

File: .github/pull_request_template.md (hunk 1 above)
Comment (PeculiarGoat) [reply .github/pull_request_template.md:1]: Direct link to card added.
AB#XXXX will autolink from the ADO card to Github, but to link from Github to the ADO card we need to put in the actual link.

File: .github/pull_request_template.md (hunk 4 above)
Comment (PeculiarGoat) [reply .github/pull_request_template.md:33]: I'll remove that for now.  I think we should formalise the `scripts` at some point though and add tests.

**Review by curtisadams28 — APPROVED** [review]
LGTM 👍

**Review by brynyShift72 — COMMENTED** [review]
Also LGTM, but might need to change or record different things after discussion

**Comment by PeculiarGoat** [comment]
@stajs happy to merge?

File: .github/pull_request_template.md (hunk 1 above)
Comment (stajs) [reply .github/pull_request_template.md]: Hmm, go with this for now, but it's that the wrong way around. [GitHub auto links AB#XXXX within PR titles/descriptions](https://docs.microsoft.com/en-us/azure/devops/boards/github/link-to-from-github?view=azure-devops#use-ab-mention-to-link-from-github-to-azure-boards-work-items).

And, it used to work fine:

image
image

But looks like someone has broke our integration 😱 - will follow up in Slack.

**Pull request merged**